lib/matplotlib/bezier.py

```diff
@@ -125,9 +125,10 @@ def find_bezier_t_intersecting_with_closedpath(bezier_point_at_t,
     end_inside = inside_closedpath(end)
 
     if not xor(start_inside, end_inside):
```

Comment:
It looks like they are and I agree that the `==` formulation is equivalent and clearer